services/simcore/docker-compose.yml

```diff
@@ -93,6 +100,13 @@ services:
       placement:
         constraints:
           - node.labels.simcore==true
+      resources:
+        reservations:
+          cpus: "0.1"
+          memory: "128M"
+        limits:
+          cpus: "0.2"
+          memory: "512M"
```

Comment:
for this guy, we might also need to run a for loop on the publicAPI and see what is happening. 0.2CPU is very low isn't it?

Reply:
As discussed in whereby